web/src/main/java/com/epam/esm/web/controller/UserController.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.esm.web.controller;
+
+import com.epam.esm.domain.dto.UserDTO;
+import com.epam.esm.service.UserService;
+import com.epam.esm.web.util.HateoasProvider;
+import com.epam.esm.web.util.PaginationProvider;
+import org.springframework.hateoas.PagedModel;
+import org.springframework.web.bind.annotation.*;

```

Comment:
* is a bad practice

Reply:
thanks, was too used to an IDE simplification. replaced